### PR TITLE
Add a page for HTMLMediaElement.preservesPitch

### PR DESCRIPTION
**Pull request by wbamberg**
This comes out of https://github.com/mdn/content/issues/3783#issuecomment-1074513476.

We should also add `spec_url` to the BCD.

### CODE REVIEW

**Comment by wbamberg** [comment]
BCD update -> https://github.com/mdn/browser-compat-data/pull/15474

File: files/en-us/web/api/htmlmediaelement/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ _This interface also inherits properties from its ancestors {{domxref("HTMLEleme
 - {{domxref("HTMLMediaElement.preload")}}
   - : Is a {{domxref("DOMString")}} that reflects the {{htmlattrxref("preload", "video")}} HTML attribute, indicating what data should be preloaded, if any. Possible values are: `none`, `metadata`, `auto`.
 - {{domxref("HTMLMediaElement.preservesPitch")}}
-  - : Is a {{jsxref('Boolean')}} that determines if the pitch of the sound will be preserved. If set to `false`, the pitch will adjust to the speed of the audio. This is implemented with prefixes in Firefox (`mozPreservesPitch`) and WebKit (`webkitPreservesPitch`).
+  - : Is a {{jsxref('Boolean')}} that determines if the pitch of the sound will be preserved. If set to `false`, the pitch will adjust to the speed of the audio..
```
Comment (teoli2003) [inline files/en-us/web/api/htmlmediaelement/index.md:79]: It doesn't return the wrapper object but a boolean value (We have plenty of these errors with DOMString and Boolean all over the place, we fix them little by little for the moment)

```suggestion
  - : Is a boolean value that determines if the pitch of the sound will be preserved. If set to `false`, the pitch will adjust to the speed of the audio.
```

File: files/en-us/web/api/htmlmediaelement/preservespitch/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+By default this property is `true` (the pitch is adjusted).
+
+## Syntax
+
+```js
+// video
+video.preservesPitch = false;
+// audio
+audio.preservesPitch = false;
+```
```
Comment (teoli2003) [inline files/en-us/web/api/htmlmediaelement/preservespitch/index.md:24]: We don't have a _Syntax_ section any more for properties. (See the [template](https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_property_subpage_template))

```suggestion
```

File: files/en-us/web/api/htmlmediaelement/preservespitch/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+### Value
```
Comment (teoli2003) [inline files/en-us/web/api/htmlmediaelement/preservespitch/index.md:26]:
```suggestion
## Value
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/htmlmediaelement/preservespitch/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  - HTML DOM
+  - HTMLMediaElement
+  - Property
+browser-compat: api.HTMLMediaElement.preservesPitch
+---
+{{APIRef("HTML DOM")}}
+
+The **`HTMLMediaElement.preservesPitch`** property determines whether or not the browser should adjust the pitch of the audio to compensate for changes to the playback rate made by setting {{domxref("HTMLMediaElement.playbackRate")}}.
+
+
```
Comment (teoli2003) [inline files/en-us/web/api/htmlmediaelement/preservespitch/index.md:15]:
```suggestion
```

I'm removing an extraneous empty line.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**